src/modules/home/HomePage.js

```diff
@@ -15,15 +16,21 @@ import FontAwesome from 'react-fontawesome';
 import './index.css';
 import Config from '../../config';
 import { hashHistory } from 'react-router';
-import appSession$ from './../../modules/app/appSession';
 import Logos from '../../components/logos/Logos';
 
-const reportURL = Config.magasin.urls.client.report.goToReport(appSession$);
-const magasinURL = Config.magasin.urls.client.magasin.goToMagasin(appSession$);
-const analysisURL = Config.magasin.urls.client.analysis.addAnalysis(appSession$);
+const reportURL = s => Config.magasin.urls.client.report.goToReport(s);
```

Comment:
what i am saying is:

```
const reportURL = Config.magasin.urls.client.report.goToReport;
```

AND

```
const reportURL = s => Config.magasin.urls.client.report.goToReport(s);
```

is the exact same thing. And the first one is better because  it doesnt introduce code paths.